qa-include/pages/admin/admin-categories.php

```diff
@@ -218,16 +218,35 @@
 					$recalc = $hassubcategory && $inslug !== $editcategory['tags'];
 				}
 
+				qa_report_event('cat_edit', qa_get_logged_in_userid(), qa_get_logged_in_handle(), qa_cookie_get(), array(
+					'categoryid' => $editcategory['categoryid'],
+					'parentid' => @$inparentid,
```

Comment:
Any chance we can get rid of the `@`? It is just hiding errors. I'm not seeing this in context (only the added lines because I'm so lazy!) but checking with `empty()` or `isset()` is usually the classic `@`-removal pattern.

Reply:
I agree, using `isset` is better than `@`